components/index/CardsReference/Variant/Chart.vue

```diff
@@ -61,6 +61,7 @@
         />
       </template>
     </scrollable-chart>
+    <slot name="additionalButton" />
```

Comment:
```suggestion
    <template #additionalButton>
      <slot name="additionalButton" />
    </template>
```

これってお作法的にtemplateで囲む必要があるはずなんですよね。

Reply:
@kaizumaki 
レビューありがとうございます。
親コンポーネント(@/components/index/_shared/DataView.vue)の名前付きスロットにコンテンツを指定しているわけではないので、templateを使用していません。  
大変恐縮ですが、templateでslotを囲うべきという推奨ルール等あればリンク等教えていただけないでしょうか？

Reply:
お、すみません、ああーなるほど `DataView.vue`に `additionalButton` というslotはないのですね。てっきりあるのかと思ってました。私の勘違いでした 🙇‍♀️  このままで大丈夫です！